brambling/forms/organizer.py

```diff
@@ -265,6 +266,30 @@ def _post_clean(self):
             self._update_errors(e)
 
 
+class CustomFormForm(forms.ModelForm):
+    class Meta:
+        model = CustomForm
+        exclude = ('event',)
```

Comment:
If there's no current use for them, it's going to confuse users. We should definitely exclude it IMO.